docs/documentation/server_admin/topics/users/con-aia.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// server_admin/topics/users.adoc
+
+[id="con-aia_{context}"]
+= Application initiated actions
+
+Application initiated actions (AIA) allow client application to request user to perform some action on {project_name} side. Usually when OIDC client application
+wants user to login, it redirects him to the login URL as described in the <<con-oidc_{context}, OIDC section>>. After login, user is redirected back to the client application.
+He needs to optionally perform the actions, which were added to his account by administrator as described in the <<proc-setting-required-actions_{context}, previous section>>
+and then being redirected back to the application immediately. However AIA allows client application to request some required actions from the user during login. This can be
```

Comment:
```suggestion
and then is immediately redirected back to the application. However, AIA allows the client application to request some required actions from the user during login. This can be
```